### ISSUE TITLE
Stopping the browser from opening the html file when a report is generated 

### ISSUE THREAD

**Issue by harrywer** [clinthuffman/PAL#28]
Hi.

Is there anyway of stopping a browser window from opening the html file when a report is generated ?
We want to generate the reports for our 100 servers in batch (which I have already done).

Regards,

Harry W.

**Comment by clinthuffman** [comment]
Yes, there is an undocumented parameter that you can pass into the PAL.ps1 script. Set $DisplayReport to $False. Thanks for reminding me of it. I can add it to the PAL Wizard interface.

For example:
start /LOW /WAIT Powershell -ExecutionPolicy ByPass -NoProfile -File ".\PAL.ps1" -Log "C:\Users\clint\Desktop\SamplePerfmonLog.blg" -ThresholdFile "C:\Users\clint\Desktop\PALPro\PALWizard\bin\Debug\Common.xml" -Interval "AUTO" -IsOutputHtml $True -HtmlOutputFileName "[LogFileName]_PAL_ANALYSIS_[DateTimeStamp].htm" -IsOutputXml $False -XmlOutputFileName "[LogFileName]_PAL_ANALYSIS_[DateTimeStamp].xml" -AllCounterStats $False -NumberOfThreads 4 -IsLowPriority $True -PhysicalMemory "16" **-DisplayReport $False**

**Comment by clinthuffman** [comment]
Leaving this issue open until added to the PAL Wizard UI.
